packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkEncodingEndToEnd.spec.ts

```diff
@@ -152,7 +152,7 @@ describe("End to end chunked encoding", () => {
 		);
 
 		const root = flexTree.root;
-		assert(root.is(numberSequenceRootSchema.rootFieldSchema));
+		assert(root.isExactly(numberSequenceRootSchema.rootFieldSchema));
```

Comment:
What is the long term plan for this layer? Will it use the kernel directly (plus schema)?

Reply:
chunked encoding will not use nodeKernel. Node kernel depends on view schema, so it can't be required for encoding trees since its possible to open a document and summarize it without making a simple-tree view.

I do not expect major changes to the chunk encoding code: only its tests use flex tree (flex tree is a view schema so the same logic applies to why it can't use flex tree).

If you are specifically asking about the chunked encoding end to end tests, what it uses is pretty arbitrary based on how "end" is defined. I don't think it matters to much. I don't think it will use kernel directly as that only intended for implementing the simple tree API, which is not something these tests do.